### PR TITLE
TEET-1421 fixes

### PR DESCRIPTION
**Pull request by outamaa**
PR fixes bugs in cost item form validation

### CODE REVIEW

File: app/frontend/src/cljs/teet/asset/cost_items_view.cljs [hunk 1]
```diff
@@ -62,57 +63,72 @@
...
-               (tr [:asset :validate :min-value] {:min-value min-value-by-ref})
-
-               (and max-value-by-ref (> n max-value-by-ref))
-               (tr [:asset :validate :max-value] {:max-value max-value-by-ref})))))
-
-       ;; no validation otherwise
-       nil))))
+      (or (when (and (string? v)
+                     (re-matches only-whitespace-pattern v))
+            "Empty spaces, what are we living for?")
```
Comment (tatut) [inline app/frontend/src/cljs/teet/asset/cost_items_view.cljs:81]: 🎵 should this be localized?

File: app/frontend/src/cljs/teet/asset/cost_items_view.cljs [hunk 2]
```diff
@@ -62,57 +63,72 @@
...
+                     (not (re-matches integer-pattern v)))
+                (tr [:asset :validate :integer-format])
+
+                (and (= valueType :db.type/bigdec)
+                     (not (re-matches decimal-pattern v)))
+                (tr [:asset :validate :decimal-format])
+
+                (and (= valueType :db.type/bigdec)
+                     (too-many-decimal-digits? v))
+                (tr [:asset :validate :decimal-precision] {:precision maximum-decimal-precision}) ;; TODO proper error message
```
Comment (tatut) [inline app/frontend/src/cljs/teet/asset/cost_items_view.cljs:112]: is this improper? fix TODO

File: app/frontend/src/cljs/teet/asset/cost_items_view.cljs (hunk 1 above)
Comment (outamaa) [reply app/frontend/src/cljs/teet/asset/cost_items_view.cljs:81]: Wow good catch...

**Review by tatut — CHANGES_REQUESTED** [review]
some localization fixes

File: app/common/resources/public/language/en.edn [hunk 3]
```diff
@@ -439,7 +441,7 @@
   :task.type/eia-pre-assessment "EIA pre-assessment",
   :task.type/owners-supervision "Owners' supervision",
   :task.group/study "Studies",
-  :file.status/draft "Draft",
+  :file.status/draft "permission-already-granted",
```
Comment (solita-erkkiv) [inline app/common/resources/public/language/en.edn:444]: Not sure if this is something that tags along from my commit, but looks strange.

**Review by solita-erkkiv — APPROVED** [review]

File: app/common/resources/public/language/en.edn (hunk 3 above)
Comment (tatut) [reply app/common/resources/public/language/en.edn:444]: looks like a mistake in localization excel

Comment (outamaa) [reply app/common/resources/public/language/en.edn:444]: @solita-erkkiv remove it from the Excel, it'll be removed from the localizations on the next import.

**Pull request merged**